arm-relay/2017-04-01/swagger/relay.json

```diff
@@ -1568,13 +1595,22 @@
       "properties": {
         "location": {
           "type": "string",
+		  "x-ms-mutability": [
+            "read",
+            "create"
+          ],
           "description": "Resource location"
         },
         "tags": {
           "type": "object",
           "additionalProperties": {
             "type": "string"
           },
+          "x-ms-mutability": [
```

Comment:
is there a reason why you included this, please see: "When the extension is applied with all the three values; "x-ms-mutability": ["create", "read", "update"] (order of the values is not important) OR when this extension is not applied on a model property; it has the same effect in both the cases. Thus applying this extension with all the three values on all the settable properties is not required. This will ensure the spec is visibly cleaner" (from https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#rules )